nilearn/_utils/compat.py

```diff
@@ -58,3 +62,33 @@ def md5_hash(string):
         m = hashlib.md5()
         m.update(string)
         return m.hexdigest()
+
+
+def get_affine(img):
+    """A function to return affine upon nibabel version.
+
+    nibabel > 2.0.0 returns img.affine otherwise img.get_affine()
+    """
+    if isinstance(img, _basestring):
+        img = nibabel.load(img)
+
+    if LooseVersion(nibabel.__version__) > LooseVersion('2.0.0'):
+        affine = img.affine
```

Comment:
This should stay in this module. Look at how python 2 / python 3 versions are handled:

```python
if sys.version_info[0] == 3:
    # Python 3 declarations
else:
   # Python 2 declarations
```

You should do the same here:
```python
if LooseVersion(nibabel.__version__) > LooseVersion('2.0.0'):
    # Code compatible with newest versions
else:
    # Code compatible with older versions
```